src/stores/FrameStore.ts

```diff
@@ -248,6 +251,19 @@ export class FrameStore {
         return null;
     }
 
+    public getImageValueFromArcsec(arcsecValue: number): number {
```

Comment:
@kswang1029 this assumes the projection is constant, because it gets image distance from arclength based on the CDELT. Is that ok?

Reply:
Yes, that’s fine. 
In general, especially wide field images with some projection types, the angular size per pixel changes everywhere in the image. If we want to do it right, we should see a rectangle or an ellipse changing its size (but still a rectangle or an ellipse) while moving in the image. We might consider to support this but I think we need more thoughts and need input from the science team.